api/libproject/devstart.rb

```diff
@@ -823,8 +843,10 @@ def deploy(cmd_name, args, with_cron)
     : (op.opts.promote ? "--promote" : "--no-promote")
   quiet = op.opts.quiet ? " --quiet" : ""
 
-  common.run_inline %W{rm -f #{GSUITE_ADMIN_KEY_PATH}}
-  get_gsuite_admin_key(gcc.project)
+  if with_gsuite_admin
+    common.run_inline %W{rm -f #{GSUITE_ADMIN_KEY_PATH}}
+    get_gsuite_admin_key(gcc.project)
```

Comment:
Possibly a future TODO; might be nice to regenerate the key here.

Reply:
TODO added. We'll also need to think about how to safely delete old keys.